### DISCUSSION TITLE
What are the differences between plugin netbox_qrcode

### DISCUSSION THREAD

**Discussion by sk011** [netbox-community/netbox#13856] [Help Wanted!]
What are the differences between plugin 
https://github.com/k01ek/netbox-qrcode and https://gitlab.nrp-nautilus.io/nrp/netbox-qrcode/-/tree/master
First URL located in offical netbox plugin page and pypi site
Second URL located in non offical site, but has more features, for example https://gitlab.nrp-nautilus.io/nrp/netbox-qrcode/-/tree/master#printing-qr-codes

Can I trust it and how to install it. From readme not quite clear

Thanks

**Comment by DanSheps** [comment]
The nrp one is considerably newer. 

I wonder if one of the maintainers would be interested in commenting on it.
